packages/app/components/Settings/TeamsSettings.tsx

```diff
@@ -0,0 +1,97 @@
+import { API } from "@koh/api-client";
+import { UpdateProfileParams } from "@koh/common";
+import { Button, Col, Form, Input, message, Space, Switch } from "antd";
+import { pick } from "lodash";
+import styled from "styled-components";
+import React, { ReactElement } from "react";
+import useSWR from "swr";
+
+export default function TeamsSettings(): ReactElement {
+  const { TextArea } = Input;
+  const { data: profile, mutate } = useSWR(`api/v1/profile`, async () =>
+    API.profile.index()
+  );
+
+  const HeaderTitle = styled(Space)`
```

Comment:
tee bee aitch might be nice to abstract this styled component too 🤔 we just don't rly have a precedent for that... previously i would just import a styled component from whatever other .tsx file used it first, but maybe we should have like a style utils directory or somethin...? idk not the hugest deal but would be interested in hearing other opinions

Reply:
I was thinking of like a `/common` directory but _I wasn't sure_

Reply:
hmmm 🤔 I feel like keeping it close to the other settings components makes sense if it's only being used in setting components. Maybe we make a file like `components/Settings/Styled.tsx` and put the shared styled components for settings in there?